SprintingMod/manifest.json

```diff
@@ -8,5 +8,6 @@
     "Build": ""
   },
   "Description": "Allows player to sprint with (or without) stamina drain on key press and hold with the option of key toggle.",
+  "UniqueID": "Patrick.SprintingMod",
```

Comment:
Please check that the unique ID is adequate. I needed to add something to get the mod loaded as the value is now required.